nasa-gibs/worldview
# WV-2437: On mobile, category label should not show on "Featured" tab

## Description

When the "Featured" tab is chosen in a mobile layout, the header element which would display the category title in other tabs (e.g. Air Quality seen below under Hazards and Disasters tab) should not be visible.  Right now it just shows "All"

Fixes # WV-2437

## How To Test

- Click on "Add Layers"
- Click the header dropdown and selected "Featured"
- "All" should no longer be there.

<img width="516" alt="Screen Shot 2023-03-08 at 2 12 35 PM" src="https://user-images.githubusercontent.com/5401206/223863014-a974a948-e220-4f11-a0ad-81e83db7da62.png">

@nasa-gibs/worldview
